Video Player/app.js

```diff
@@ -0,0 +1,56 @@
+const player=document.querySelector(".player");
+const video=player.querySelector(".viewer");
+const progress=player.querySelector(".progress");
+const progressBar=player.querySelector(".progress__filled");
+const toggle = player.querySelector(".toggle");
+const skipButtons=player.querySelectorAll("[data-skip]");
+const ranges=player.querySelector("range");
+//player-slider: video kaydırıcı
```

Comment:
Yorumlari projeyi commit ederken sil. Eger bu sekilde not olarak birakacaksan ingilizce olarak birak. En kötü ceviriden cevir. Cünkü: 
Bir yabanci bu yorumu anlayamaz ve kötü görünür. 
Ingilizce yazmaya calismak en kötü ingilizceni gelistirir.